app/views/endemics/lamb-error-exception.njk

```diff
@@ -0,0 +1,41 @@
+{% extends './layouts/layout.njk' %}
```

Comment:
You have created a new view but not linked to it from anywhere? Has this been done separately?

Reply:
IT wasnt me I only touched content changes